Update enterprise dropdown

pmahnke opened the pull request:
## Done

Update enterprise dropdown with an array of updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/#enterprise
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1YBdQvLuqEpEQr_QqyycxMhZr4OaLapMlNJyYKCmPJsQ/edit#
)

webteam-app commented:
Starting demo at: https://ubuntu-com-canonical-web-and-design-pr-6189.run.demo.haus/

anthonydillon commented on templates/templates/_navigation-enterprise-h.html:
```diff
@@ -144,9 +144,9 @@ <h4 class="p-muted-heading"><a href="/server">Data centre&nbsp;&rsaquo;</a></h4>
         <ul class="p-text-list--small is-bordered">
           <li class="p-list__item"><a href="/kubernetes/features">Kubernetes support for AI/ML</a></li>
-          <li class="p-list__item"><a href="/blog/ai-ml-model-ubuntu">Deploy your first model with Kubeflow</a></li>
+          <li class="p-list__item"><a href="/kubeflow/install">Deploy your first model with Kubeflow</a></li>
```
This page doesn't result in a deployed model but is a much better start to kubeflow. Maybe replace the text to something like "Start using kubeflow with MicroK8s".

pmahnke replied:
ok

pmahnke commented:
@anthonydillon ok, ok, I did a _much_ bigger clean-up, changed a lot, added a lot.  I think it is a lot more up-to-date and better... happy for suggestions.

pmahnke commented:
@anthonydillon not sure we wanted to change it there? @matthewpaulthomas thoughts?
